Tools/Pipeline/Common/PipelineController.cs

```diff
@@ -866,9 +866,14 @@ private List<string> GetDirectories(string folder)
 
         public void Exclude(bool delete)
         {
-            if (delete && !View.ShowDeleteDialog(SelectedItems))
+            // We don't want to show a delete confirmation for any items outside the project folder
+            var filteredItems = new List<IProjectItem>(SelectedItems.Where(i => !i.OriginalPath.Contains("..")));
+
+            if (filteredItems.Count > 0 && delete && !View.ShowDeleteDialog(filteredItems))
```

Comment:
I did test that exact case and it works fine. It only prevents the delete operation from occurring, not the removal from the tree.

I kept it general because you could have the ".." selected along with other files/folders.